src/components/TipsPagination.vue

```diff
@@ -28,12 +28,11 @@
 </template>
 
 <script>
-import { get, isEmpty } from 'lodash-es';
+import { get, isEmpty, times } from 'lodash-es';
```

Comment:
I'd prefer not to use lodash at all

Reply:
why so? I'm using it in edge cases when js doesn't have something out of the box. Also `lodash-es` uses es6 imports so only used code will appear in the bundle. Or maybe you prefer to use ramda instead? 🙃

Reply:
I don't have a specific reason, I don't think lodash is needed really. Up to you to decide